Gravity/Gravity.NUnit.Integration/Gravity.NUnit.Integration.csproj

```diff
@@ -0,0 +1,95 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{7DF2B93B-3B14-45BB-B466-31F5D76E3BA1}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Gravity.NUnit.Integration</RootNamespace>
+    <AssemblyName>Gravity.NUnit.Integration</AssemblyName>
```

Comment:
I am really not a fan of having "NUnit" in the name of the project. I think we'd be better off if this was named Gravity.Test.Integration, or something like that - you can easily notice all the test projects by starting with Gravity.Test, and the fact that it is an NUnit test is not relevant unless we were also planning to have other types of tests

Reply:
makes sense